javascript/src/stories/components/ChannelMessages/MessageListView.stories.js

```diff
@@ -12,32 +12,11 @@ const Template = (args) => <MessageListView {...args}/>;
 
 export const WithSearchBar = Template.bind({});
 WithSearchBar.args = {
-  messages: [{
-    "type": "message",
-    "subtype": "channel_join",
-    "ts": "1594143066.000200",
-    "user": "U017218J9B3",
-    "text": "<@U017218J9B3> has joined the channel",
-    "channel": "section-6pm",
-    "user_profile": {
-      "real_name": "Test Person"
-    }
-  },
-  {
-    "type": "message",
-    "subtype": "channel_join",
-    "ts": "1594143066.000200",
-    "user": "U017218J9B3",
-    "text": "Office hours at <https://ucsb.zoom.us/j/89220034995?pwd=VTlHNXJpTVgrSEs5QUtlMDdqMC9wQT09>",
-    "channel": "section-6pm",
-    "user_profile": {
-      "real_name": "Test Person"
-    }
-  }],
+  messages: MessageFixtures.tenMessages
 };
 
 export const NoSearchBar = Template.bind({});
 NoSearchBar.args = {
-    searchField: false,
-    ...WithSearchBar.args
+  messages: MessageFixtures.tenMessages,
+  searchField: false
```

Comment:
I don't think these two lines needed to be changed, the three dots simply means that it will take in the properties from `WithSearchBar.args` so it will still be `messages: MessageFixtures.tenMessages` since that's what is in `WithSearchBar.args`.

```suggestion
    searchField: false,
    ...WithSearchBar.args
```

Reply:
I was wondering which way to write it, and I wrote it like this in case arguments for `WithSearchBar` ever got changed, but looking back I agree the three dots notation is probably better in this case